frontend/src/services/AxiosService.js

```diff
@@ -0,0 +1,16 @@
+import axios from "axios";
+
+const KonvaUrl = '/konva'
```

Comment:
not sure if necessary at all 🤔

```suggestion
const konvaUrl = '/konva'
```